tutorials/vr/vr_primer.rst

```diff
@@ -17,12 +17,21 @@ You can use the function :ref:`get_interfaces <class_ARVRServer_get_interfaces>`
 
 To enable an interface you execute the following code:
 
-::
+.. tabs::
+ .. code-tab:: gdscript GDScript 
 
     var arvr_interface = ARVRServer.find_interface("Native mobile")
     if arvr_interface and arvr_interface.initialize():
         get_viewport().arvr = true
 
+ .. code-tab:: csharp
+
+    var ArvrInterface = ARVRServer.FindInterface("Native mobile");
```

Comment:
Local variable should begin with a lower case letter, according to the standard naming convention.